src/main.cpp

```diff
@@ -174,13 +177,15 @@ void control_thread()
 void ext_mgr_thread()
 {
     uint32_t last_call = 0;
+    int thread_id_ext_mgr = sw_watchdog_register(3000);
 
     // quite long watchdog timeout as we might be dealing with slow communication (e.g. modems
     // using AT commands via serial interface)
-    int wdt_channel = watchdog_register(3000);
 
     while (1) {
-        watchdog_feed(wdt_channel);
+
+        //feeds sw_wtchdg
```

Comment:
That's exactly what the name of the function says, so this comment is not necessary.